src/components/chat/comment-textbox.tsx

```diff
@@ -12,12 +12,21 @@ interface IProps {
 const minTextAreaHeight = 35;
 
 export const CommentTextBox: React.FC<IProps> = ({ activeNavTab, numPostedComments, onPostComment }) => {
+  const textareaRef = useRef<HTMLTextAreaElement | null>(null);
   const [commentTextAreaHeight, setCommentTextAreaHeight] = useState(minTextAreaHeight);
   const [commentAdded, setCommentAdded] = useState(false);
   const [commentText, setCommentText] = useState("");
   const textareaStyle = {height: commentTextAreaHeight};
   const postButtonClass = classNames("comment-footer-button", "themed-negative", activeNavTab,
                                      { disabled: !commentAdded, "no-action": !commentAdded });
+  // resize textarea when user deletes some text
+  useEffect(() => {
+    if (textareaRef && textareaRef.current) {
+      textareaRef.current.style.height = minTextAreaHeight + "px";
```

Comment:
Unless I'm missing something, this line doesn't appear to be necessary given that the value of `textareaRef.current.style.height` is reset to a different value in line 27 below.

Reply:
When I remove the line, the textarea does not resize when I delete part of the text.

Reply:
Bizarre, but in that case add a comment to that effect because otherwise it looks like redundant code.

Reply:
Still need to add the comment about the redundant-appearing code in line 25.